examples/grpc/hello_server.cc

```diff
@@ -0,0 +1,154 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Example RPC server using gRPC.
+
+#include <cstdlib>
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include <grpcpp/grpcpp.h>
+
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_cat.h"
+#include "examples/grpc/hello.grpc.pb.h"
+#include "examples/grpc/hello.pb.h"
+#include "examples/grpc/stackdriver.h"
+#include "include/grpcpp/opencensus.h"
+#include "opencensus/exporters/stats/prometheus/prometheus_exporter.h"
+#include "opencensus/exporters/stats/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/stats/stdout/stdout_exporter.h"
+#include "opencensus/exporters/trace/stackdriver/stackdriver_exporter.h"
+#include "opencensus/exporters/trace/stdout/stdout_exporter.h"
+#include "opencensus/stats/stats.h"
+#include "opencensus/trace/sampler.h"
+#include "opencensus/trace/span.h"
+#include "opencensus/trace/trace_config.h"
+#include "prometheus/exposer.h"
+#include "src/cpp/ext/filters/census/grpc_plugin.h"
```

Comment:
Is this needed? Is this an internal header?

Reply:
Yes, this is needed for calls to GetSpanFromServerContext.